tools/devops/automation/scripts/bash/configure-build.sh

```diff
@@ -0,0 +1,10 @@
+#!/bin/bash -xe
```

Comment:
super duper minor - all the other scripts are `-ex`. I know it doesn't make it a difference. It just stood out. 😄